ceremonies/src/lib.rs

```diff
@@ -316,27 +316,52 @@ decl_module! {
     }
 }
 
-decl_event!(
-    pub enum Event<T>
-    where
-        AccountId = <T as frame_system::Config>::AccountId,
-    {
-        ParticipantRegistered(AccountId),
-    }
-);
-
 decl_error! {
     pub enum Error for Module<T: Config> {
         ParticipantAlreadyRegistered,
         BadProofOfAttendanceSignature,
-        BadAttestationSignature,
         BadAttendeeSignature,
         MeetupLocationNotFound,
         MeetupTimeCalculationError,
-        NoValidClaims
+        NoValidClaims,
+        /// only the CeremonyMaster can call this function
```

Comment:
you use this in different places. I guess we should say: "sender doesn't have the necessary authority to perform action"